services/ui-src/src/components/alerts/ErrorAlert.tsx

```diff
@@ -0,0 +1,51 @@
+// components
```

Comment:
New component with a couple variants, pretty flexible props, and a style override prop so we can handle it inline however we want.